### PR TITLE
Devex - No files should be passed over API gateway due to the 6mb limit. Instead, fetch from s3 for all files

### PR DESCRIPTION
**Pull request by rachelschneiderman**

### CODE REVIEW

File: shared/src/business/useCases/courtIssuedDocument/serveCourtIssuedDocumentInteractor.test.js [hunk 1]
```diff
@@ -24,6 +24,8 @@ describe('serveCourtIssuedDocumentInteractor', () => {
   let testPdfDoc;
   let extendCase;
 
+  const mockPdfUrl = 'http://www.therestartpage.com/#';
```
Comment (rachaelparris) [inline shared/src/business/useCases/courtIssuedDocument/serveCourtIssuedDocumentInteractor.test.js]: It was requested (I believe by Waldo) that we use `example.com` for all testing URLs.

File: web-client/src/presenter/actions/TrialSession/runTrialSessionPlanningReportAction.test.js [hunk 2]
```diff
@@ -5,21 +5,15 @@ import { runTrialSessionPlanningReportAction } from './runTrialSessionPlanningRe
...
   beforeAll(() => {
-    global.window = global;
-    global.Blob = () => {};
-    const fakeData =
-      'JVBERi0xLjEKJcKlwrHDqwoKMSAwIG9iagogIDw8IC9UeXBlIC9DYXRhbG9nCiAgICAgL1BhZ2VzIDIgMCBSCiAgPj4KZW5kb2JqCgoyIDAgb2JqCiAgPDwgL1R5cGUgL1BhZ2VzCiAgICAgL0tpZHMgWzMgMCBSXQogICAgIC9Db3VudCAxCiAgICAgL01lZGlhQm94IFswIDAgMzAwIDE0NF0KICA+PgplbmRvYmoKCjMgMCBvYmoKICA8PCAgL1R5cGUgL1BhZ2UKICAgICAgL1BhcmVudCAyIDAgUgogICAgICAvUmVzb3VyY2VzCiAgICAgICA8PCAvRm9udAogICAgICAgICAgIDw8IC9GMQogICAgICAgICAgICAgICA8PCAvVHlwZSAvRm9udAogICAgICAgICAgICAgICAgICAvU3VidHlwZSAvVHlwZTEKICAgICAgICAgICAgICAgICAgL0Jhc2VGb250IC9UaW1lcy1Sb21hbgogICAgICAgICAgICAgICA+PgogICAgICAgICAgID4+CiAgICAgICA+PgogICAgICAvQ29udGVudHMgNCAwIFIKICA+PgplbmRvYmoKCjQgMCBvYmoKICA8PCAvTGVuZ3RoIDg0ID4+CnN0cmVhbQogIEJUCiAgICAvRjEgMTggVGYKICAgIDUgODAgVGQKICAgIChDb25ncmF0aW9ucywgeW91IGZvdW5kIHRoZSBFYXN0ZXIgRWdnLikgVGoKICBFVAplbmRzdHJlYW0KZW5kb2JqCgp4cmVmCjAgNQowMDAwMDAwMDAwIDY1NTM1IGYgCjAwMDAwMDAwMTggMDAwMDAgbiAKMDAwMDAwMDA3NyAwMDAwMCBuIAowMDAwMDAwMTc4IDAwMDAwIG4gCjAwMDAwMDA0NTcgMDAwMDAgbiAKdHJhaWxlcgogIDw8ICAvUm9vdCAxIDAgUgogICAgICAvU2l6ZSA1CiAgPj4Kc3RhcnR4cmVmCjU2NQolJUVPRgo=';
+    const fakeUrl = 'www.heyYouCoolCatsAndKittens';
```
Comment (rachaelparris) [inline web-client/src/presenter/actions/TrialSession/runTrialSessionPlanningReportAction.test.js]: One more URL here to update. (Also, I hope this is a real site.)

**Review by rachaelparris — APPROVED** [review]
Looks great!

**Review by codyseibert — APPROVED** [review]

File: web-client/src/presenter/actions/TrialSession/runTrialSessionPlanningReportAction.js [hunk 3]
```diff
@@ -10,21 +10,16 @@ import { state } from 'cerebral';
...
       term,
       year,
     });
 
-  const pdfFile = new Blob([planningReportPdf], { type: 'application/pdf' });
-
-  const pdfUrl = router.createObjectURL(pdfFile);
-
-  return { pdfUrl };
+  return { pdfUrl: planningReportPdfUrl.url };
```
Comment (kkoskelin) [inline web-client/src/presenter/actions/TrialSession/runTrialSessionPlanningReportAction.js:24]: Does `planningReportPdfUrl` really have a property named 'url'?

Comment (kkoskelin) [reply web-client/src/presenter/actions/TrialSession/runTrialSessionPlanningReportAction.js:24]: Gosh, I guess it does -- my bad.  Perhaps the variable naming just needs some tweaking.

Comment (rachelschneiderman) [reply web-client/src/presenter/actions/TrialSession/runTrialSessionPlanningReportAction.js:24]: Good point, it's a little confusing! I will rename this and include that change in the next PR.

**Pull request merged**